src/PrestaShopBundle/Resources/config/admin/routing_international.yml

```diff
@@ -0,0 +1,22 @@
+admin_international_translations_list:
+    path: /translations/list
+    methods:  [POST]
+    defaults:
+        _controller: PrestaShopBundle:Admin/Translations:list
+        _legacy_controller: AdminTranslations
+        theme: classic
```

Comment:
I'm wondering.. why do you need both ``theme`` and ``type`` defaults properties, did I miss something ?